README.adoc

```diff
@@ -407,34 +419,13 @@ to execute before the test cases. The [hotspot=setup file=0]`setup()` method ret
-that a `ConstraintViolationException` exception is thrown when one of the method-level 
-constraints is violated. A call with an incorrect parameter, [hotspot=incorrectCode file=0]`incorrectCode`, 
-is made to the [hotspot=launchSpacecraft file=1]`launchSpacecraft()` method of the [hotspot=Spacecraft file=1]`Spacecraft` 
-bean. The method returns `false`, which violates the defined constraint, and a `ConstraintViolationException` 
-exception is thrown. The exception includes the constraint violation message, which is 
-[hotspot=actualResponse file=0]`must be true` in this example.
+* The [hotspot=testNoFieldLevelConstraintViolations file=0]`testNoFieldLevelConstraintViolations()` test case verifies that constraint violations do not occur when valid data is supplied to the [hotspot=Astronaut file=0]`Astronaut` and [hotspot=Spacecraft file=0]`Spacecraft` bean attributes.
+
+* The [hotspot=testFieldLevelConstraintViolation file=0]`testFieldLevelConstraintViolation()` test case verifies that the appropriate constraint violations occur when data that is supplied to the `Astronaut` and `Spacecraft` attributes violates the defined constraints. Because 3 constraint violations are defined, 3 `ConstraintViolation` objects are returned as a set from the [hotspot=Response file=0]`validate` call. The 3 expected messages are [hotspot=expectedDestinationResponse file=0]`"must be greater than 0"` for the negative distance specified in the destination map, [hotspot=expectedEmailResponse file=0]`"must be a well-formed email address"` for the incorrect email address, and the custom [hotspot=expectedSerialNumberResponse file=0]`"serial number is not valid"` message for the serial number.
+
+* The [hotspot=testNoMethodLevelConstraintViolations file=0]`testNoMethodLevelConstraintViolations()` test case verifies that the method-level constraints that are specified on the [hotspot=launchSpacecraft file=1]`launchSpacecraft()` method of the [hotspot=Spacecraft file=1]`Spacecraft` bean are validated when the method is called with no violations. In this test, the call to the [hotspot=launchSpacecraft file=0]`launchSpacecraft()` method is made with the [hotspot=OpenLiberty file=0]`OpenLiberty` argument. A value of `true` is returned, which passes the specified constraints.
+
+* The [hotspot=testMethodLevelConstraintViolation file=0]`testMethodLevelConstraintViolation()` test case verifies that a `ConstraintViolationException` exception is thrown when one of the method-level constraints is violated. A call with an incorrect parameter, [hotspot=incorrectCode file=0]`incorrectCode`, is made to the [hotspot=launchSpacecraft file=1]`launchSpacecraft()` method of the [hotspot=Spacecraft file=1]`Spacecraft` bean. The method returns `false`, which violates the defined constraint, and a `ConstraintViolationException` exception is thrown. The exception includes the constraint violation message, which is [hotspot=actualResponse file=0]`must be true` in this example.
```

Comment:
The exception includes the constraint violation message, which is [hotspot=actualResponse file=0]`must be true` in this example.
--->
The exception includes the constraint violation message, which in this example is [hotspot=actualResponse file=0]`must be true` .